src/www/ui/api/Helper/UploadHelper.php

```diff
@@ -821,6 +821,9 @@ public function getUploadLicenseList($uploadId, $agents, $printContainers, $bool
         $responseList[] = $responseRow->getArray();
       }
     }
-    return $responseList;
+    $Offset = $Page * $Limit;
```

Comment:
Please use proper naming convention for variables. They should follow camel casing.